doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1106,8 +1107,10 @@ Response:
       + Members
           + keyValues - when used, the request payload uses the `keyValues` simplified entity
             representation. See "Simplified Entity Representation" section for details.
+          + upsert - when used, entity is updated if already exits (otherwise this situation causes
+            a 422 Unprocessable Entity error).
```

Comment:
clarify this better, perhaps putting that sentence separate

Reply:
Trying to clarify in 6566b8aad31afa25fed44b9b8a5aec3661ef71f1